src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -12,6 +12,7 @@ CA1830 | Performance | Info | PreferTypedStringBuilderAppendOverloads, [Document
 CA1831 | Performance | Warning | UseAsSpanInsteadOfRangeIndexerAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1831)
 CA1832 | Performance | Info | UseAsSpanInsteadOfRangeIndexerAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1832)
 CA1833 | Performance | Info | UseAsSpanInsteadOfRangeIndexerAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1833)
+CA1834 | Performance | Info | PreferConstCharOverConstUnitStringAnalyzer, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1834)
```

Comment:
Would be nice to create a ticket on https://github.com/MicrosoftDocs/visualstudio-docs/issues to add the documentation for this analyzer.

Reply:
Thanks @Evangelink.
@mavasani sent us all detailed instructions today on how to document all of our new rules.

Reply:
@pgovind I just realized I'm using the same ruleID as you, but you submitted your PR first, so I'll bump up mine by one.